# Information specific to each force sensor.
# Readout is in Newtons
int32 heel
int32 inner
int32 outer
int32 heelR
int32 innerR
int32 outerR
